tests/helpers/mod.rs

```diff
@@ -7,6 +7,7 @@ use actix_web::test;
 use anyhow::{anyhow, Error};
 use serde_json::Value;
 
+pub mod api_assertions;
 pub mod app;
 pub mod db;
 pub mod http_client;
```

Comment:
nit: add `pub type RumbaTestResponse = ServiceResponse<EitherBody<EitherBody<BoxBody>>>;` and we can use this instead of this long type.